src/api/XRSessionWithLayer.ts

```diff
@@ -94,6 +94,8 @@ export class XRSessionWithLayer {
 
 					gl.bindFramebuffer(gl.FRAMEBUFFER, this.tempFramebuffer)
 					const existingClearColor = gl.getParameter(gl.COLOR_CLEAR_VALUE)
+					const existingFrameBuffer = gl.getParameter(gl.FRAMEBUFFER_BINDING)
+
```

Comment:
Can you explain a little the framebuffer changes? Why are we switching to using different bindings for the clear and then for rendering all the layers? (At least, how I'm reading it - I'm not sure if I'm reading it correctly)

Reply:
The old code changed the current framebuffer.
If there was code that relied on it staying there, it would be broken.